src/routes/docs/advanced/platform/+layout.svelte

```diff
@@ -152,6 +152,11 @@
                     new: isNewUntil('28 Feb 2025'),
                     label: 'Abuse',
                     href: '/docs/advanced/platform/abuse'
+                },
+                {
+                    new: isNewUntil('31 July 2025'),
+                    label: 'Refund Policy',
+                    href: '/docs/advanced/platform/refund-policy'
```

Comment:
Since this section is all about policies, we don't need to include "policy" here.

```suggestion
                    new: isNewUntil('31 July 2025'),
                    label: 'Refund',
                    href: '/docs/advanced/platform/refund'
```

Make sure to update the file name too.